game.libretro.scummvm/addon.xml

```diff
@@ -13,8 +10,11 @@
                library_windx="game.libretro.scummvm.dll"
                library_wingl="game.libretro.scummvm.dll">
         <platforms></platforms>
-        <extensions></extensions>
+        <extensions>scummvm|scum</extensions>
         <supports_vfs>false</supports_vfs>
+        <supports_game_loop>true</supports_game_loop>
+        <supports_standalone>true</supports_standalone>
```

Comment:
Yes, that's essential. [ScummVM has a UI in which you have to add your games before you can run them](https://upload.wikimedia.org/wikipedia/commons/c/c6/ScummVM_Windows_Screenshot.png). A game is usually a directory with different files depending on the emulated game engine. Once you added them, you can either run them from the in-game menu, or create `.scummvm` files to directly launch into a game from Kodi. ScummVM defines a unique game ID for all supported games. This ID needs to be put into the file, and that way it knows which game it should launch. Sounds complicated, but you get used to it quite quickly.